poetry.lock

```diff
@@ -1754,6 +1912,16 @@ files = [
     {file = "wcwidth-0.2.10.tar.gz", hash = "sha256:390c7454101092a6a5e43baad8f83de615463af459201709556b6e4b1c861f97"},
 ]
 
+[[package]]
+name = "wireless"
```

Comment:
this package is not being maintained anymore. Is there any other alternative for this package?

https://pypi.org/project/wireless/

Reply:
I think there's a newer package out that does the same thing. I'll edit and push

Reply:
great!!